index.html

```diff
@@ -163,6 +166,65 @@ <h1 class="clr-blue">About me</h1>
       </p>
       <button class="btn btn-2">Get my resume</button>
     </section>
-    <section></section>
+    <ul class="skills-div">
+      <li class="skills center"></li>
+      <div class="skill">
+        <img src="imgs/p1.png" alt="skills" class="p1" />
+        <li><h1 class="s">Languages</h1></li>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">HTML</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">CSS</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Javascript</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+      <li class="skills center frameworks"></li>
+      <div class="skill">
+        <img src="imgs/p2.png" alt="skills" class="p1" />
+        <li><h1 class="s">Frameworks</h1></li>
+        <ul class="li li-3">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">React.js</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">RSpec</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <br />
+        <ul class="li li-3">
+          <li class="li-2">Capybara</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Selenium</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+      <li class="skills center"></li>
+      <div class="skill">
+        <img src="imgs/p3.png" alt="skills" class="p1" />
+        <li><h1 class="s">Skills</h1></li>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Database Management</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Version Control</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <br />
+        <ul class="li">
+          <li class="li-2">CLI</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Web Development</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">API Design</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </ul>
```

Comment:
I think the previous reviewers confused you regarding this so let me explain clearly how much I can.And from them I am sorry 😭 

Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on Rails</li>
          <li>Rspec</li>
          <li>Capybara</li>
          <li>Selenium</li>
        </ul>
      </li>
      <li>
        Skills
        <ul>
          <li>Database Management</li>
          <li>Version Control</li>
          <li>CLI</li>
          <li>API Design</li>
          <li>Web Development</li>
        </ul>
      </li>
    </ul>
```

This is a general structure please consider updating the technologies as presented in Figma.